tests/console/docker.pm

```diff
@@ -28,14 +28,19 @@ use testapi;
 use utils;
 use strict;
 use registration;
+use version_utils qw(is_sle is_leap);
 
 sub run {
     select_console("root-console");
 
     install_docker_when_needed;
 
-    # boo#1072367 - docker engine doesn't have seccomp support
-    assert_script_run('docker info | tee /dev/tty | grep seccomp');
+    if (is_sle('<15') || is_leap('<15.0')) {
+        record_soft_failure('boo#1072367 - docker engine doesnt have seccomp support') unless (script_run('docker info | tee /dev/tty | grep seccomp'));
```

Comment:
You should invert logic, script_run returns bash exit code, which 0 on success. In perl condition it's opposite.
`s/unless/if/`